### DISCUSSION TITLE
UVtools window stretched or not in correct aspect ratio

### DISCUSSION THREAD

**Discussion by jrvillalobos** [sn4k3/UVtools#908] [General]
I cannot figure out whymy screen is stretched. this supposed to be circle. It affects on putting support. Please help. thank you
![Screenshot 2024-07-27 030228](https://github.com/user-attachments/assets/c532fcc9-4003-4d07-b5fb-e7fddfcf383c)

**Comment by sn4k3** [comment]
That's normal and correct, nothing wrong with it.

**Short answer:** Your printer uses unequal pixel size of 19x24um (A odd ratio), as so, to print a circle you must send a oval image to printer. To print a square you must send a rectangle image.
**Long answer:** https://github.com/sn4k3/UVtools/discussions/866#discussioncomment-9098556

**Reply by jrvillalobos** [reply]
Thank you so much for your reply. Previously, I was able to achieve the correct ratio, so I’m wondering if the issue is related to the recent update. The problem arises when adding supports: as shown in the attached picture, the left and right sides require more support, while the top and bottom do not. After adding support to the sides, the print turns out correctly, but the top and bottom still have overhangs. I need additional support in those areas, but I can’t see this need in the UVTools window, which indicates that the support is adequate.



![Screenshot 2024-07-27 131440](https://github.com/user-attachments/assets/50c3ef0f-db42-4e41-8d1f-6badd8c57017)

**Reply by sn4k3** [reply]
> Previously, I was able to achieve the correct ratio, so I’m wondering if the issue is related to the recent update

Again, there are no problem with it, you have the correct ratio. Slicer always export in correct ratio which is informed on bottom right of UVtools as `Pixel: 00x00um`. Different printers will have different ratios.

> The problem arises when adding supports: as shown in the attached picture, the left and right sides require more support

This is true, since your pixel is narrow on X direction 19 vs 24um. UVtools supports are placed as pixel value un-aware of ratio, also circles have only a radius parameter, what you need is an oval shape with wider X than Y to counter the problem but UVtools doesn't offer that option.
To tackle that you can use the circle but give it a wider diameter to account the narrow X, just multiply your regular pixel value by 1.263x to account this, you will still have a narrow support in X direction but it will compensate for it.

> After adding support to the sides, the print turns out correctly, but the top and bottom still have overhangs.

Trying to fix every issue UVtools show, most of the time is a loss of time and not required task. You need to analise, sometimes it's not critical. The sensibility of detection can be adjusted, by default it's conservative.
Looking at your image I would say it will print just fine.

**Reply by jrvillalobos** [reply]
Ok, thanks for the help. I added support on top and the bottom and it prints fine.
